examples/memory_basic_usage.py

```diff
@@ -26,24 +26,23 @@ def expensive_computation(data, column_index=0):
     return data[column_index]
 
 
-rng = np.random.RandomState(42)
-data = rng.randn(int(1e5), 10)
-tic = time.time()
+data = np.random.randn(int(1e5), 10)
```

Comment:
I actually think that seeding was genuinely useful: people forget to seed, and than the data is different at each run, and they complain that Memory doesn't work.

It would be useful to add a comment explaining why it's important above.